src/Bot/CustomCommandType.hs

```diff
@@ -0,0 +1,33 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+-- | Moved out of CustomCommand to break dependency cycle:
+--   Help depends on this type, but custom commands needs
+--   to refresh Help and therefore also this type.
```

Comment:
Do I understand correctly that this is a "haskell doc string" and it will show up in the documentation when we try to generate it?

Reply:
Yes, like javadoc.

You can run `cabal haddock exec:HyperNerd` to make it.